src/main/java/User.java

```diff
@@ -0,0 +1,23 @@
+import java.io.Serializable;
+public class User implements Serializable{
+    private String username;
+    private String password;
+    private String email;
+    boolean verified = false;
+    boolean online = false;
+    public User(String username, String password, String email){
+        this.username = username;
+        this.password = password;
+        this.email = email;
+    }
+    public String getEmail(){
```

Comment:
It's good that you're using getters and private variables to hide the implementation details.